lib/playground.dart

```diff
@@ -21,6 +21,7 @@ import 'core/keys.dart';
 import 'core/modules.dart';
 import 'dart_pad.dart';
 import 'documentation.dart';
+import 'localstorage.dart';
```

Comment:
nit: mind alphabetizing these while you're in here? It looks like they were ordered at some point in the past.